### DISCUSSION TITLE
<script setup> 新创建的组件合变量， 热更新失效

### DISCUSSION THREAD

**Discussion by fujunfei** [vuejs/rfcs#499] [General]
<script setup>  新创建的组件合变量， 热更新失效，我必须重启项目才会正常显示； 如果是在setup函数里就没这个问题，在github搜到有类似的问题，但都解决办法

**Comment by KimYangOfCat** [comment]
反馈bug或者寻求使用上的帮助，你应该去[vuejs/core](https://github.com/vuejs/core)仓库的讨论区，而不是这里。

 For feedback on bugs or help with usage, you should go to the discussion in the [vuejs/core](https://github.com/vuejs/core) repository, not here.
